src/layout/Header/MobileMenu/index.tsx

```diff
@@ -40,6 +45,98 @@ const MobileMenu: React.FC<Props> = ({ activeMobileMenu, handleMobileMenu }) =>
     setOpenTransactions(false)
   }
 
+  const location: any = useLocation()
+
+  const mainLinks = [
+    {
+      link: MENU_LINK.dashboard,
+      icon: Dashboard,
+      title: t('header.dashboard'),
+      id: 'dashboard',
+      isActive: location?.pathname?.startsWith(MENU_LINK.dashboard)
+    },
+    {
+      link: MENU_LINK.swap,
+      icon: Swap,
+      title: t('header.swap'),
+      id: 'swap',
+      isActive: location?.pathname?.startsWith(MENU_LINK.swap)
+    },
+    {
+      link: MENU_LINK.buy,
+      icon: Buy,
+      title: t('header.buy'),
+      id: 'buy',
+      isActive: location?.pathname?.startsWith(MENU_LINK.buy)
+    },
+    {
+      link: MENU_LINK.pool,
+      icon: Pool,
+      title: `${t('header.pool')} & ${t('header.farm')}`,
+      id: 'pool',
+      isActive: location?.pathname?.startsWith(MENU_LINK.pool)
+    },
+    {
+      link: `${MENU_LINK.stake}/0`,
+      icon: Stake,
+      title: t('header.stake'),
+      id: 'stake',
+      isActive: location?.pathname?.startsWith(`${MENU_LINK.stake}/`)
+    },
+    {
+      link: MENU_LINK.stakev2,
+      icon: Stake,
+      title: `${t('header.stake')} V2`,
+      id: 'stakev2',
+      isActive: location?.pathname?.startsWith(MENU_LINK.stakev2)
+    },
+
+    {
+      link: MENU_LINK.vote,
+      icon: Vote,
+      title: t('header.vote'),
+      id: 'vote',
+      isActive: location?.pathname?.startsWith(MENU_LINK.vote)
+    },
+    {
+      link: MENU_LINK.airdrop,
+      icon: Airdrop,
+      title: 'Airdrop',
+      id: 'airdrop',
+      isActive: location?.pathname?.startsWith(MENU_LINK.airdrop)
+    }
+  ]
+
+  const pangolinLinks = [
+    {
+      link: ANALYTICS_PAGE,
+      icon: ChartsIcon,
+      title: t('header.charts'),
+      id: 'charts'
+    },
+    {
+      link: 'https://gov.pangolin.exchange',
+      icon: GovernanceIcon,
+      title: t('header.forum'),
+      id: 'forum'
+    }
+  ]
+
+  const otherLinks = [
+    {
+      link: 'https://bridge.avax.network/',
+      icon: BridgeIcon,
+      title: `Avalanche ${t('header.bridge')}`,
+      id: 'bridge'
+    },
+    {
+      link: 'https://satellite.axelar.network/',
+      icon: BridgeIcon,
+      title: `Satellite ${t('header.bridge')}`,
+      id: 'satellite-bridge'
+    }
+  ]
```

Comment:
we are repeating many of the things. we should have a way to reuse it.

for e.g. may be in one file ( `menuItems.ts` in Header folder ) you can keep separate objects like this 
```js
export const bridge = {
  link: "https://bridge.avax.network/",
  icon: BridgeIcon,
  title: `Avalanche ${t("header.bridge")}`,
  id: "bridge",
};

export const satelliteBridge = {
  link: 'https://satellite.axelar.network/',
  icon: BridgeIcon,
  title: `Satellite ${t('header.bridge')}`,
  id: 'satellite-bridge'
}
```
and then in this file you just use it
```js
const otherLinks = [bridge, satelliteBridge]
```

in that way there will not be any duplication for links and labels. 
this is just an idea, if you have any better idea i am up for it.

Reply:
I've moved out the common static data. But  have kept the _title_ and _isActive_ keys same due to use of hooks.